packaging/windows/variables.wxi

```diff
@@ -26,7 +26,7 @@
   <?if $(var.Platform)=x86?>
   <?define Program_Files="ProgramFilesFolder"?>
   <?define Win64AttributeValue=no?>
-  <?define UpgradeCode="70A1576F-63B6-4659-9E39-25C7B769DDE5"?>
+  <?define UpgradeCode="25CAE344-22A6-422D-942E-942687C56A45"?>
```

Comment:
@robmen Probably missed communicated in my previous message regarding GUID for x64 (I did not create a new GUID for x64 now). I meant to say when I initially started writing (in Dec '15) wxs files for x64 I created new GUID and I did not remember doing something similar for x86. Hence I updated the GUID for x86.